### ISSUE TITLE
NTR: Procollagen peptide high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8303]
New term request Procollagen peptide high in blood (elthat_121130120215):

**Comment by pnrobinson** [comment]
@MickeySegal
what is the disease context? There are several procollagen peptides.

**Comment by MickeySegal** [comment]
The GeneReviews article has new authors, and now refers to:
https://www.ncbi.nlm.nih.gov/books/NBK587318/
"Levels of bone formation markers, such as procollagen type 1 aminoterminal propeptide (P1NP), alkaline phosphatase, or osteocalcin, are elevated in both sclerosteosis and van Buchem disease. "

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000749
- label: Elevated circulating type I procollagen aminoterminal propeptide concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: PMID:22480789|PMID:23074140
- def: The concentration of type I procollagen aminoterminal propeptide in the blood circulation is above the upper limit of normal. The aminoterminal propeptide of type I procollagen (PINP) in serum is a sensitive indicator of the synthesis of type I collagen.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8303
- synonyms:
